Exiled.API/Features/Map.cs

```diff
@@ -514,6 +515,34 @@ public static DoorVariant GetDoorByName(string doorName)
             return nameExtension == null ? null : nameExtension.TargetDoor;
         }
 
+        /// <summary>
+        /// Changes the color of a MTF unit.
+        /// </summary>
+        /// <param name="index">The index of the unit color you want to change.</param>
+        /// <param name="color">The new color of the Unit.</param>
+        public static void ChangeUnitColor(int index, string color)
+        {
+            var unit = Respawning.RespawnManager.Singleton.NamingManager.AllUnitNames[index].UnitName;
+
+            Respawning.RespawnManager.Singleton.NamingManager.AllUnitNames.Remove(Respawning.RespawnManager.Singleton.NamingManager.AllUnitNames[index]);
+            Respawning.NamingRules.UnitNamingRules.AllNamingRules[Respawning.SpawnableTeamType.NineTailedFox].AddCombination($"<color={color}>{unit}</color>", Respawning.SpawnableTeamType.NineTailedFox);
+
+            foreach (var ply in Player.List.Where(x => x.ReferenceHub.characterClassManager.CurUnitName == unit))
+            {
+                if (unit.Contains("<color"))
+                {
+                    System.Text.RegularExpressions.Regex regex = new System.Text.RegularExpressions.Regex("<color=(.*)>(.*)</color>");
+                    var v = regex.Match(unit);
+                    string s = v.Groups[1].ToString();
+                    ply.ReferenceHub.characterClassManager.NetworkCurUnitName = unit.Replace(s, color);
+                }
+                else
+                {
+                    ply.ReferenceHub.characterClassManager.NetworkCurUnitName = $"<color={color}>{unit}</color>";
+                }
```

Comment:
Simplify the regex to `<[^>]*?>` and call `Regex::Replace(string, string, string)` instead to remove tags (https://regex101.com/r/9Wsd6l/1)